client/src/styles/leafletComponents/resultItem.less

```diff
@@ -63,4 +63,9 @@
 
     .c-result-item__sharing {
       padding-top: 7px;
-    }
\ No newline at end of file
+    }
+
+.fa {
+  padding: 15px;
+  padding-top: 22px;
+}
```

Comment:
You don't want to do this.  This will impact every font-awesome icon on the page, such as the magnifying glass on the search.  Be extremely cautious when making changes that have a big scope.  Typically it is better to decorate a specific item.